db/mongoose.js

```diff
@@ -0,0 +1,11 @@
+// Registry Connection
+var mongoose = require('mongoose');
+
+mongoose.connect(process.env.MONGODB_URL, {
+    useNewUrlParser: true,
```

Comment:
Yes, because otherwise the console will generate the following warnings:
![warn](https://user-images.githubusercontent.com/37256767/74817153-75e29b00-5322-11ea-9679-b50c6b409316.png)